Khan/wonder-blocks
# 1/4 [feature:popover] WB-603: Implement PopoverContentCore

## Summary

Renders popover content and allows to create custom content.

### NOTE: Stacked PRs

This is part of [WB-579: Implement Popovers](https://khanacademy.atlassian.net/browse/WB-579)

These PRs are created to cover the whole Popover implementation:

**1. Implement PopoverContentCore (current PR): Renders popover content.**
2. Implement PopoverContent: Add text-only, icon and illustration variations.
3. Implement Popover: Adds Popover component to handle popover triggering and positioning. Also, integrates TooltipTail and TooltipPopper.
4. Popover: Add Focus management

## Implementation

### Popover
[Implementation Spec](https://khanacademy.atlassian.net/wiki/spaces/FRONTEND/pages/115310606/Popover+v1+Implementation+Spec#PopoverContentCore-public-)
- Created `wonder-blocks-popover` package
- Added `PopoverContentCore` component to be used as the base window to display content.

### Tooltip
- Added `Placement` type as part of the public API.

## Testplan
1. Open https://deploy-preview-433--wonder-blocks.netlify.com/#!/PopoverContentCore/0
2. Check the component is displaying properly.
![popover-content-core](https://user-images.githubusercontent.com/843075/59939368-bc939480-9425-11e9-990e-48ef3b45c286.png)
